tox.ini

```diff
@@ -6,6 +6,6 @@ deps =
     pip >= 6
     -rtest_requirements.txt
 commands =
-    coverage run --source pytest_asyncio -m py.test
+    coverage run --source pytest_asyncio -m py.test -n 4
```

Comment:
Why not use `-n auto`? It is described [here](https://github.com/pytest-dev/pytest-xdist#speed-up-test-runs-by-sending-tests-to-multiple-cpus) and imo should be the default choice unless there's a reason to fix the parallelism.

Reply:
Good to know about the Travis issue. Glad there's a solution 🙂